test/clients.go

```diff
@@ -35,14 +35,14 @@ import (
 
 // KubeClient holds instances of interfaces for making requests to kubernetes client.
 type KubeClient struct {
-	Kube *kubernetes.Clientset
+	Kube kubernetes.Interface
```

Comment:
I wonder if it'll be better to anonymous aggregate it.

Reply:
tbh I think we should delete this resource 100%

Reply:
that works too.